### ISSUE TITLE
[Feature Request] Show EXIF metadata in image properties

### ISSUE THREAD

**Issue by corporateshark** [telegramdesktop/tdesktop#25392]
### Is your feature request related to a problem?

Uncompressed (not recompressed by Telegram) images usually contain useful metadata (exposure time, F-number, ISO, camera info, copyright, etc).

### Describe the solution you'd like

Add a "Metadata..." option to the right-button context menu when browsing images.

### Describe alternatives you've considered

As of now, it seems to be possible only via external tools.

### Additional context

#2874

**Comment by Aokromes** [comment]
server deletes exif.

**Comment by Aokromes** [comment]
such feature belongs to https://bugs.telegram.org/

**Comment by corporateshark** [comment]
> server deletes exif.

I've just tried to send an image and unchecked the box "Compress images" - it seems all the EXIF metadata is still in place after that.

**Comment by john-preston** [comment]
No, if sent as a file, server doesn't modify the file. It could be done, although the feature request really has low priority, sorry.

**Comment by corporateshark** [comment]
Is it possible to have just something very simple, such as the attached image file resolution, exposure time, F-number, ISO, lens/camera info?
